File: Week1 JavaScript/exercise-one.js
// Write function avg which calculates average of numbers in given list.

function find_average(array) {
  var sumTotal = array.reduce(function(a, b) {
    return a + b;
  }, 0);
  
   var arrayLength = array.length;
   var average = sumTotal / arrayLength;
   
   return average;
   
}

// Read about array.reduce() method

File: Week1 RoR/exercise-one.rb
# Write function avg which calculates average of numbers in given list.

def average_array(array)
  
  if array.length < 1
   return 0
  else 
    arraySum = array.inject(0) { |sum,x| sum + x }
    arrayLength = array.length
    arrayAverage = arraySum / arrayLength
  end 
end

average_array([17, 16, 16, 16, 16, 15, 17, 17, 15, 5, 17, 17, 16])

# Read about array.inject method
